taker-frontend/src/types.ts

```diff
@@ -235,6 +235,7 @@ export enum StateGroupKey {
 export interface CfdOrderRequestPayload {
     order_id: string;
     quantity: number;
+    position: string;
```

Comment:
I think I'd have a slight preference for a 2nd endpoint instead of adding a field here.

Reply:
I don't think we need 2 endpoints at all - the `order_id` unambiguously denotes the order. different positions will need different order_ids.

Reply:
yep I don't see the need, the position is in the payload.